apps/src/code-studio/pd/foorm/editor/form/FormEditorManager.jsx

```diff
@@ -130,8 +129,8 @@ class FormEditorManager extends React.Component {
 
   updateFormData = formData => {
     this.props.setFormData(formData);
```

Comment:
NIT: i would like to advocate for getting into the habit of updating arrow functions to regular functions (bound if necessary) whenever you touch them. personally i like to do that for all the functions in any component i touch.